elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -260,6 +255,7 @@ class PfeContentSet extends PFElement {
     // Validate that the light DOM data exists before building
     if (this.hasValidLightDOM) {
       this._build();
+      if (!this.isIE11) this.render();
```

Comment:
So by using the delayRender option, we can manually trigger the render based on if it's IE or not.  Very cool!